openslides/motions/static/templates/motions/motion-detail.html

```diff
@@ -258,102 +258,62 @@ <h3 ng-if="motion.polls.length > 0" translate>Voting result</h3>
             <i class="fa fa-bar-chart fa-lg"></i>
             <translate>New vote</translate>
           </button>
+
+          <button ng-click="createChangeRecommendation.startCreating()" class="btn btn-default btn-sm"
+                  ng-if="motion.isAllowed('can_manage') && viewChangeRecommendations.mode == 'original'">
+            <i class="fa fa-plus fa-lg"></i>
+            <translate>Change Recommendation</translate>
+          </button>
       </div>
     </div>
   </div>
 </div>
 
 <div class="details">
   <div class="row">
-  <div class="linenumber-toolbar">
-    <!-- inline editing -->
-    <div class="pull-right inline-editing-activator"
-         ng-if="motion.isAllowed('update') && version == motion.getVersion(-1).id">
-      <button ng-if="!inlineEditing.active" ng-click="inlineEditing.enable()" class="btn btn-sm btn-default">
-        <i class="fa fa-pencil-square-o"></i>
-        <translate>Inline editing</translate>
-      </button>
-      <button ng-if="inlineEditing.active" ng-click="inlineEditing.disable()" class="btn btn-sm btn-default">
-        <i class="fa fa-times-circle"></i>
-        <translate>Inline editing</translate>
-      </button>
-    </div>
 
-    <!-- line number mode -->
-    <div class="line-number-setter {{ lineNumberMode }}">
-      <span class="btn-group" data-toggle="buttons">
-        <div class="btn btn-sm btn-default disabled">
-          <i class="fa fa-list-ol" aria-hidden="true"></i>
-          <translate>Line numbering</translate>:
-        </div>
-        <label class="btn btn-sm btn-default" ng-class="{active: (lineNumberMode == 'none')}"
-               ng-click="lineNumberMode = 'none';">
-          <input type="radio" name="lineNumberMode" value="none" ng-model="lineNumberMode"
-                 ng-checked="lineNumberMode == 'none'">
-          <translate>none</translate>
-        </label>
-        <label class="btn btn-sm btn-default" ng-class="{active: (lineNumberMode == 'inline')}"
-               ng-click="lineNumberMode = 'inline'">
-          <input type="radio" name="lineNumberMode" value="inline" ng-model="lineNumberMode"
-                 ng-checked="lineNumberMode == 'inline'">
-          <translate>inline</translate>
-        </label>
-        <label class="btn btn-sm btn-default" ng-class="{active: (lineNumberMode == 'outside')}"
-               ng-click="lineNumberMode = 'outside'">
-          <input type="radio" name="lineNumberMode" value="outside" ng-model="lineNumberMode"
-                 ng-checked="lineNumberMode == 'outside'">
-          <translate>outside</translate>
-        </label>
-      </span>
+    <!-- Motion toolbar -->
+    <ng-include src="'static/templates/motions/motion-detail-toolbar.html'"></ng-include>
```

Comment:
Proposal: Move all parts of motion detail to a subdirectory static/templates/motions/motion-detail/toolbar.html. So also the templates directory is much more cleaner.